docs/guides/implementation-guides/ide-support.md

```diff
@@ -95,6 +95,19 @@ Alternatively you can use `Design.DataContext` attached property. As well as `De
 </Window>
 ```
 
+### Setting the Design-Time DataContext in Code
+
+Declaring the design-time `DataContext` in XAML will not work if the view model's only constructor has parameters. This happens with typical [dependency injection patterns](https://docs.avaloniaui.net/docs/guides/implementation-guides/how-to-implement-dependency-injection). One option is to use a special [design-time data model](https://docs.avaloniaui.net/docs/guides/implementation-guides/how-to-use-design-time-data) with a parameterless constructor.  However, a design-time `DataContext` that references a view model constructor with parameters may be set in code with the `Design.SetDataContext` method.
```

Comment:
The first line is a bit awkward. Consider: "Setting the design-time `DataContext` in XAML directly is only possible if the ViewModel has a parameterless constructor." It's not strictly correct, but creating a custom `MarkupExtension` to use a service locator and construct the correct VM is complicated for docs.

Should change the links to be rooted or relative. eg. `[text...](/docs/guides/implementation-guides/how-to-use-design-time-data)` or '[text...](how-to-use-design-time-data)`

This perhaps should be in how-to-use-design-time-data instead, but up to you / other reviewers.